ui/src/pages/AllStories.tsx

```diff
@@ -8,6 +8,7 @@ import * as React from "react";
 import styled from "styled-components";
```

Comment:
I'm not entirely sure what this is doing but do you link the all stories page to this view story page here or is it something else?

Reply:
If not we'll also need to add this logic in a separate pr

Reply:
yes when u click an entry in the table it opens the view story page for that entry